tests/test_run_reference_vmm.py

```diff
@@ -19,20 +23,24 @@ def process_exists(pid):
         return True
 
 
-@pytest.mark.parametrize("kernel", KERNELS)
+def resource_path(resource_type, resource_name):
+    return os.path.abspath(os.path.join(
+        os.path.dirname(os.path.realpath(__file__)),
+        "..",
+        "resources/{}/{}".format(resource_type, resource_name)
+    ))
+
+
+@pytest.mark.parametrize("kernel", KERNELS_INITRAMFS)
 def test_reference_vmm(kernel):
-    """Start the reference VMM and trust that it works."""
+    """Start the reference VMM and verify that it works."""
 
     # Memory config
     mem_size_mib = 1024
 
     # Kernel config
     cmdline = "console=ttyS0 i8042.nokbd reboot=k panic=1 pci=off"
```

Comment:
nit: can we update the `cmdline` with the suggestion from [here](https://github.com/rust-vmm/vmm-reference/pull/41#discussion_r534318525)?